### ISSUE TITLE
Style issue - key details hidden/truncated from dashboard

### ISSUE THREAD

**Issue by avoerman** [cribeiro84/azure-devops-pull-request-hub#32]
Key details like the branch name and code repo are being truncated because the fist column is flexed equally with the other three. Seems like that column could be a higher flex value, or the date column could have a max-width since there's a lot of white-space present.

![image](https://user-images.githubusercontent.com/645331/66211066-3f64f600-e689-11e9-9dd5-913c6862693f.png)

**Comment by cribeiro84** [comment]
Thanks a lot @avoerman for the inputs and feedback, really appreciated. I did not see those issues before and I will try to simulate it on my machine. Would you mind to share a few more details?

 - What is your browser and its version?
 - Operating System?
 - Scree Resolution 

Thanks again :-)

**Comment by hanneswidrig** [comment]
The browser/OS/screen resolution do not matter in this case, similar issues are visible on the screenshots for the extension.

![image](https://user-images.githubusercontent.com/3399497/66318603-a2ed5e80-e8e9-11e9-9e39-c348db65c46b.png)

Below is an example of another PR Dashboard available for Azure DevOps. It's obviously not designed with Microsoft Fabric UI in mind, however it might provide some inspiration of better information prioritization.

![image](https://user-images.githubusercontent.com/3399497/66319548-4ab75c00-e8eb-11e9-80ae-9d73ae41f76c.png)

The repository which the PR is for is clear and concise.

**Comment by hanneswidrig** [comment]
If you're looking for some ideas about restructuring the PR row UI, I could mock something up.

**Comment by avoerman** [comment]
> Thanks a lot @avoerman for the inputs and feedback, really appreciated. I did not see those issues before and I will try to simulate it on my machine. Would you mind to share a few more details?
> 
> * What is your browser and its version?
> * Operating System?
> * Scree Resolution
> 
> Thanks again :-)

Browser: Chrome Version 76.0.3809.132 (Official Build) (64-bit)
OS: Windows 10 Enterprise
Resolution: 2560 x 1440 scaled to 125%. Measured browser window at 2030x1014. Browser zoom is standard 100%.

**Comment by hanneswidrig** [comment]
@cribeiro84 I hope to have a pretty major update in PR this week, here is a very nearly complete UI.
![PR Dashboard](https://user-images.githubusercontent.com/3399497/66725689-904fb980-ee02-11e9-9f06-06c20b89f574.png)

If there's any fields that you absolutely want to stay, please let me know. "Work items" and reviewer section is not done yet, still working on that.

**Comment by cribeiro84** [comment]
> @cribeiro84 I hope to have a pretty major update in PR this week, here is a very nearly complete UI.
> ![PR Dashboard](https://user-images.githubusercontent.com/3399497/66725689-904fb980-ee02-11e9-9f06-06c20b89f574.png)
> 
> If there's any fields that you absolutely want to stay, please let me know. "Work items" and reviewer section is not done yet, still working on that.

Wow! This looks awesome, thanks a lot @hanneswidrig. 

About the Work Items icon, I tried to add it to the UI but even though the API shows the property called "workItemRefs", it's always null or empty, it may require an additional REST call to capture the linked work items to the PR. The same I was trying to add the "Comments" icon but no success too. :-(

The only field I would like to keep is the **Duration** where it calculates in Days, hours and minutes since the PR was created - with that, I can simply check the PRs older than X days and abandon them for my team.

**Comment by hanneswidrig** [comment]
Yeah you have to use the other REST Clients available from Microsoft's API. I got work Items populated and comments is the next thing I plan on doing. I have a ton of other ideas too, but this will be a good start.

I will add the duration specifically right of where the relative date is now.

**Comment by cribeiro84** [comment]
@hanneswidrig I really appreciate it! It will look amazing!

**Comment by hanneswidrig** [comment]
Wanted to give you an update @cribeiro84 , I am almost done with the resign and feature set, however I have need to do some reorganization so it might still be a little while.

**Comment by cribeiro84** [comment]
Hello @hanneswidrig, I really thought I did reply to you, so sorry about the delay. Totally fine with that, I will wait it then so I can try to fix again some issues found in the filters.

Once again, thanks a lot for your help on this. Looks awesome.

**Comment by hanneswidrig** [comment]
I wanted to give an update.

 I decided I just wanted to make much larger changes; essentially rewriting the extension from scratch. It'll be a hard fork of your project, if you are planning to add features to your PR Hub, don't wait for my changes. You could expect that I probably won't be done before Christmas.

**Comment by cribeiro84** [comment]
Hey @hanneswidrig, thanks for the updates, is there a way to help you on your fork so I could also help on the rewriting it? I know that the code is pretty far from a good shape and I was planning to have a refactoring on the upcoming features/bugfixes as in parallel as was learning more about React which I did not have any experience before.

Thank you very much

**Comment by hanneswidrig** [comment]
I appreciate the offer, but I'd rather work on it myself. It's more than a minor refactor.

**Comment by cribeiro84** [comment]
Hello @hanneswidrig, no problem. I just worked on some UI improvements and bugfixes to close this issue and in parallel we can keep your fork with the UI redesign for a new milestone. What do you think?

Hi @avoerman,
A new version is available with the style issues fix. Hope this version provides a better experience.

Thank you both!

**Comment by hanneswidrig** [comment]
Good improvements, if you were curious how I am rewriting the extension, you can check my repo below.

https://github.com/hanneswidrig/old-azure-devops-pr-hub

**Comment by cribeiro84** [comment]
> Good improvements, if you were curious how I am rewriting the extension, you can check my repo below.
> 
> https://github.com/hanneswidrig/old-azure-devops-pr-hub

Thanks a lot @hanneswidrig, I did it already and looks awesome the new code structure, I'm very curious to see it running.

**Comment by avoerman** [comment]
> Hello @hanneswidrig, no problem. I just worked on some UI improvements and bugfixes to close this issue and in parallel we can keep your fork with the UI redesign for a new milestone. What do you think?
> 
> Hi @avoerman,
> A new version is available with the style issues fix. Hope this version provides a better experience.
> 
> Thank you both!

Thanks - it's working much better for our team now.
